kotlinpoet/build.gradle.kts

```diff
@@ -37,19 +39,61 @@ kotlin {
     withJava()
   }
 
+  js {
+    browser {
+      testTask {
+        useKarma {
+          useChromeHeadless()
+        }
+      }
+    }
+    nodejs {
+      testTask {
+        useMocha()
+      }
+    }
+    binaries.library()
+
+  }
+
+  @OptIn(ExperimentalWasmDsl::class)
+  wasmJs {
+    browser {
+      testTask {
+        useKarma {
+          useChromeHeadless()
+        }
+      }
+    }
+    nodejs {
+      testTask {
+        useMocha()
+      }
+    }
+    binaries.library()
+  }
+
   @OptIn(ExperimentalKotlinGradlePluginApi::class)
   compilerOptions {
     allWarningsAsErrors.set(true)
     optIn.add("com.squareup.kotlinpoet.DelicateKotlinPoetApi")
+    freeCompilerArgs.add("-Xexpect-actual-classes")
```

Comment:
According to https://youtrack.jetbrains.com/issue/KT-61573, this feature is stable since 1.9.20, so this shouldn't be needed?

Reply:
https://youtrack.jetbrains.com/issue/KT-61573 seems to mean that `expect/actual fun` has stabilized, but `expect/actual classes` is still in Beta.

You can also see in https://kotlinlang.org/docs/components-stability.html#current-stability-of-kotlin-components that the Status of `expected and actual classes` is still Beta.

The `expect/actual` classes will also show warnings if you try to remove this option, both in the IDE and when executing the gradle command for compilation.

Reply:
Ah, you're right, thanks for the clarification!